Client/src/Components/product-overview/StyleSelector.jsx

```diff
@@ -24,6 +26,7 @@ function StyleSelector({ setStyleIndex }) {
     // purpose: provide sizes for selector
     Object.keys(styles[styleIndex].skus).forEach((key) => {
```

Comment:
Why not use map here?
const sizes = Object.keys(...).map(..)

Reply:
When I tried to refactor to use map I'm getting errors related to an unfulfilled promise. I am taking a note to revisit to refactor at a later time.